src/types/typeobject.rs

```diff
@@ -127,6 +130,16 @@ pub trait PyTypeMethods<'py>: crate::sealed::Sealed {
     fn is_subclass_of<T>(&self) -> PyResult<bool>
     where
         T: PyTypeInfo;
+
+    /// Return Python mro
+    ///
+    /// Equivalent to the Python expression `__mro__`.
+    fn mro(&self) -> PyResult<Bound<'_, PyTuple>>;
```

Comment:
Here we need to feed the `'py` lifetime too. If we use `'_`, it compiles, but the lifetime actually used is the implicit lifetime in `&'_ self`, which is much shorter than `'py`.

```suggestion
    fn mro(&self) -> PyResult<Bound<'py, PyTuple>>;
```